src/camera/camera3d.jl

```diff
@@ -555,7 +569,6 @@ function update_cam!(scene::Scene, camera::Camera3D, area3d::Rect)
     bb = FRect3D(area3d)
     width = widths(bb)
     half_width = width/2f0
-    lower_corner = minimum(bb)
```

Comment:
You might also want to remove `lower_corner = minimum(bb)` in `src/camera/old_camera3d.jl`